topydo.conf

```diff
@@ -46,3 +46,6 @@ append_parent_contexts      = 0
 ; context_color = magenta
 ; metadata_color = green
 ; link_color = light-cyan
+
+[aliases]
+;showall = ls -x
```

Comment:
My suggestion is to add all variants of `lsprj` and `lscon` as a "pre-installed" alias (and remove the variants from the `_SUBCOMMAND_MAP`).